flink-docs/pom.xml

```diff
@@ -173,6 +173,19 @@ under the License.
             <version>${project.version}</version>
         </dependency>
 
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-sql-gateway</artifactId>
+			<version>${project.version}</version>
+		</dependency>
+
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-sql-gateway</artifactId>
+			<version>${project.version}</version>
+			<type>test-jar</type>
```

Comment:
Which part of the test jar do we need?

Reply:
`DocumentingSqlGatewayRestEndpoint` is located at test jar of Sql Gateway module.